### ISSUE TITLE
No module named cheroot.server

### ISSUE THREAD

**Issue by friscoMad** [alexa-pi/AlexaPi#185]
**Your OS (including version) where you are running AlexaPi:**
```
OSMC (Debian 8.7)
```

**Your hardware platform and model you are running on:**
```
Raspberry PI 2
```

**Python release (`python2 --version`):**
```
Python 2.7.9
```

**Description of problem:**
```
Setup did not finish, it halts after setting client info
```

**Expected:**
```
Setup completes correctly
```

**Problem-relevant `config.yaml` entries:**
```

```

**Steps to reproduce:**

1. 
2. 
3. 

**Traceback (if applicable):**
```
Traceback (most recent call last):
  File "./auth_web.py", line 8, in <module>
    import cherrypy
  File "/usr/local/lib/python2.7/dist-packages/cherrypy/__init__.py", line 76, in <module>
    from cherrypy import _cprequest
  File "/usr/local/lib/python2.7/dist-packages/cherrypy/_cprequest.py", line 10, in <module>
    from cherrypy import _cpreqbody, _cpconfig
  File "/usr/local/lib/python2.7/dist-packages/cherrypy/_cpreqbody.py", line 135, in <module>
    import cheroot.server
ImportError: No module named cheroot.server
```

**Additional info:**
```
I did choose the default options except for airplay that I did choose not to install.

The first run it did not work due to missing build-essentials on my system.
The second everything went ok, except this error.
Just in case, I did run the script a third time with the same error, and all python dependencies seem to be installed:

> pip 1.5.6 from /usr/lib/python2.7/dist-packages (python 2.7)
> Requirement already satisfied (use --upgrade to upgrade): requests>=2.4.3 in /usr/lib/python2.7/dist-packages (from -r ./requirements.txt (line 1))
> Requirement already satisfied (use --upgrade to upgrade): CherryPy>=4.0.0 in /usr/local/lib/python2.7/dist-packages (from -r ./requirements.txt (line 2))
> Requirement already satisfied (use --upgrade to upgrade): python-vlc from git+https://github.com/oaubert/python-vlc.git in /usr/local/lib/python2.7/dist-packages (from -r ./requirements.txt (line 3))
> Requirement already satisfied (use --upgrade to upgrade): pyalsaaudio>=0.8.2 in /usr/local/lib/python2.7/dist-packages (from -r ./requirements.txt (line 4))
> Requirement already satisfied (use --upgrade to upgrade): webrtcvad>=2.0.7 in /usr/local/lib/python2.7/dist-packages (from -r ./requirements.txt (line 5))
> Requirement already satisfied (use --upgrade to upgrade): pyyaml in /usr/local/lib/python2.7/dist-packages (from -r ./requirements.txt (line 6))
> Requirement already satisfied (use --upgrade to upgrade): pocketsphinx>=0.1.3 in /usr/local/lib/python2.7/dist-packages (from -r ./requirements.txt (line 7))
> Requirement already satisfied (use --upgrade to upgrade): coloredlogs in /usr/local/lib/python2.7/dist-packages (from -r ./requirements.txt (line 8))

```

**Comment by renekliment** [comment]
Thank you for the report. It seems that CherryPy==4.0.0 doesn't have `cheroot` in requirements. I'll bump the requirements (probably in general) and I'll let you know to test. Thanks!

**Comment by renekliment** [comment]
@frisco82 See #187. If you could test the PR, that would be great. Thank you!

**Comment by renekliment** [comment]
Fixed in `dev`. I'll close this when it gets merged to `master`.

**Comment by cgioconda** [comment]
Having the same issue as well - when do you anticipate dev will be merged to master?

**Comment by renekliment** [comment]
@cgioconda Some time this week. Until Friday hopefully.

**Comment by renekliment** [comment]
Merged into `master` in version 1.5. Closing.
